nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -908,17 +924,37 @@ def from_tensorflow(self, graph):
         params : dict
             A dict of name: tvm.nd.array pairs, used as pretrained weights
         """
-        # Parse throught all nodes and start extracting
-        # params aka Const nodes
-        # input nodes  : First const node
-        # normal nodes : other normal nodes
 
         try:
             from tensorflow.python.framework import tensor_util
         except ImportError as e:
             raise ImportError(
                 "Unable to import tensorflow which is required {}".format(e))
 
+        # Parse the nodes briefly to determine the preconditions:
```

Comment:
Brief parsing is a good addition, Thanks. It helps to list all missing ops at one place.
Please make it a separate function instead of inline.